config/README.md

```diff
@@ -0,0 +1,145 @@
+# Running the workflow on your own samples
```

Comment:
Instead of the copy I would welcome a move of the section, with a pointer from the main readme to this one. 
Would that work as well?

Reply:
Thanks @dominikburri I was looking at this and wonder if I should move the whole README in the config , as each of the sections are essential for the appropriate run of the workflow. Is there some other rule for putting the README in the root path?

Reply:
If you open the github page, the README content is displayed. I guess if this is not there anymore, the start site would look quite empty. But not quite sure about any rules or so.

Reply:
Hi @dominikburri I agree with your statement. Since we want these instructions both in the config and the home page, I just made a README.md in the config with a link to the main README file. The softlink alternative does not work in github so I suggest we keep this option for now. Can you please review this solution?